Sources/Sentry/SentrySwizzle.m

```diff
@@ -136,9 +139,14 @@ + (BOOL)swizzleInstanceMethod:(SEL)selector
                          mode:(SentrySwizzleMode)mode
                           key:(const void *)key
 {
-    NSAssert(!(NULL == key && SentrySwizzleModeAlways != mode),
+    NSAssert(!(key == NULL && mode != SentrySwizzleModeAlways),
```

Comment:
@armcknight mentioned in #2109 that this should be a `NSParameterAssert`, but that can't have a description with it. So I think  `NSAssert` is probably better in the end.

Reply:
TIL NSParameterAssert is just an NSAssert with a predefined message
```
#define NSParameterAssert(condition) NSAssert((condition), @"Invalid parameter not satisfying: %@", @#condition)
```